src/tests/unit/tests/injected/adapters/resolution-creator.test.ts

```diff
@@ -59,16 +59,15 @@ describe('ResolutionCreator', () => {
         expect(actual).toEqual(expected);
     });
 
-    it.each`
-        testId                     | expectedText
-        ${'aria-input-field-name'} | ${"Inspect the element using the Accessibility pane in the browser Developer Tools to verify that the field's accessible name is complete without its associated <label>."}
-        ${'color-contrast'}        | ${"If the text is intended to be invisible, it passes.\nIf the text is intended to be visible, use Accessibility Insights for Windows (or the Colour Contrast Analyser if you're testing on a Mac) to manually verify that it has sufficient contrast compared to the background. If the background is an image or gradient, test an area where contrast appears to be lowest.\nFor detailed test instructions, see Assessment > Text legibility > Contrast."}
-        ${'link-in-text-block'}    | ${"Manually verify that the link text EITHER has a contrast ratio of at least 3:1 compared to surrounding text OR has a distinct visual style (such as underlined, bolded, or italicized). To measure contrast, use Accessibility Insights for Windows (or the Colour Contrast Analyser if you're testing on a Mac)."}
-        ${'th-has-data-cells'}     | ${'Examine the header cell in the context of the table to verify that it has no data cells.'}
-        ${'bogus test id'}         | ${"No 'How to check' guidance has been supplied.  Please contact the Accessibility Insights team."}
-    `('outputs correct check resolution with id=$testId', ({ testId, expectedText }) => {
+    it.each([
```

Comment:
You don't need this many test cases now since the work here is really just making sure the end product has the passed in rule id.

Reply:
ah very true